packages/styling/src/ParticipantView/ParticipantView-layout.scss

```
@@ -106,38 +106,39 @@ $scope-name: 'str-video__participant-view';
     }
   }
 
-  .str-video__participant-details__connection-quality {
+  .str-video__connection-quality-indicator {
     position: absolute;
     bottom: 0;
     right: 0;
 
-    border-radius: var(--str-video__border-radius-sm) 0
-      var(--str-video__border-radius-sm) 0;
+    inline-size: var(--str-video__size-32);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)